Error: Not working on react-native v0.41.2

"react-native-maps": "^0.12.4"

I have an error when build **IOS** version:
```
** BUILD FAILED **


The following build commands failed:

	CompileC /Users/development/projects/app-client/ios/build/Build/Intermediates/AirMaps.build/Debug-iphonesimulator/AirMaps.build/Objects-normal/x86_64/AIRMapCallout.o AirMaps/AIRMapCallout.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
```

Build for **Android** was successful, but when application launched:
```
03-09 03:16:06.828 2684-2718/com.appclient E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #3
    Process: com.appclient, PID: 2684
    java.lang.RuntimeException: An error occurred while executing doInBackground()
        at android.os.AsyncTask$3.done(AsyncTask.java:309)
        at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
        at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
        at java.util.concurrent.FutureTask.run(FutureTask.java:242)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
        at java.lang.Thread.run(Thread.java:818)
    Caused by: java.lang.NoClassDefFoundError: com.google.android.gms.maps.GoogleMapOptions
        at com.airbnb.android.react.maps.AirMapManager.<init>(AirMapManager.java:54)
        at com.airbnb.android.react.maps.MapsPackage.createViewManagers(MapsPackage.java:39)
        at com.facebook.react.XReactInstanceManagerImpl.createAllViewManagers(XReactInstanceManagerImpl.java:699)
        at com.facebook.react.CoreModulesPackage.createUIManager(CoreModulesPackage.java:206)
        at com.facebook.react.CoreModulesPackage.access$200(CoreModulesPackage.java:71)
        at com.facebook.react.CoreModulesPackage$8.get(CoreModulesPackage.java:145)
        at com.facebook.react.CoreModulesPackage$8.get(CoreModulesPackage.java:142)
        at com.facebook.react.LazyReactPackage.createNativeModules(LazyReactPackage.java:76)
        at com.facebook.react.XReactInstanceManagerImpl.processPackage(XReactInstanceManagerImpl.java:958)
        at com.facebook.react.XReactInstanceManagerImpl.createReactContext(XReactInstanceManagerImpl.java:863)
        at com.facebook.react.XReactInstanceManagerImpl.access$600(XReactInstanceManagerImpl.java:110)
        at com.facebook.react.XReactInstanceManagerImpl$ReactContextInitAsyncTask.doInBackground(XReactInstanceManagerImpl.java:214)
        at com.facebook.react.XReactInstanceManagerImpl$ReactContextInitAsyncTask.doInBackground(XReactInstanceManagerImpl.java:193)
        at android.os.AsyncTask$2.call(AsyncTask.java:295)
        at java.util.concurrent.FutureTask.run(FutureTask.java:237)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588) 
        at java.lang.Thread.run(Thread.java:818) 
```

Has anyone solved this problem?


**UPD:**
It is working with react-native-maps@v.0.13.0
For **IOS** build was successful, but for **Android** need to add in **android/app/build.gradle**:
```
compile "com.google.android.gms:play-services-maps:10.2.0"
```

--- a/lib/android/build.gradle
+++ b/lib/android/build.gradle
@@ -35,6 +35,6 @@ android {
 
 dependencies {
   provided "com.facebook.react:react-native:+"
-  compile "com.google.android.gms:play-services-base:10.0.1"
-  compile "com.google.android.gms:play-services-maps:10.0.1"
+  compile "com.google.android.gms:play-services-base:10.2.0"
+  compile "com.google.android.gms:play-services-maps:10.2.0"
 }

